backend/backends/__init__.py

```
@@ -708,13 +762,55 @@ def get_tts_backend_for_engine(engine: str) -> TTSBackend:
+def drop_tts_backend_for_engine(engine: str) -> bool:
+    """Unload and forget one TTS backend instance so Python can release VRAM refs."""
+    global _tts_backends
+    with _tts_backends_lock:
+        backend = _tts_backends.pop(engine, None)
+    if backend is None:
+        return False
+    try:
+        backend.unload_model()
+        return True
+    except Exception:
+        logger.exception("Failed to unload TTS backend %s", engine)
```

Ruff (0.15.12)

[error] Undefined name `logger`

(F821)